src/NSwag.AspNetCore/SwaggerUiSettingsBase.cs

```diff
@@ -35,9 +35,7 @@ public abstract class SwaggerUiSettingsBase : SwaggerSettings
         public SwaggerUiSettingsBase()
         {
             TransformToExternalPath = (internalUiRoute, request) =>
-            {
-                return request.GetBasePath() + internalUiRoute;
-            };
+                request.GetBasePath()?.TrimEnd('/') + '/' + internalUiRoute?.TrimStart('/');
```

Comment:
Shouldnt the produced path start with a /?

Reply:
This will providing `GetBasePath` returns a path that starts with `/` or returns `null` or `string.Empty`. I guess another question would be should `GetBasePath` always start with `/`?

Reply:
If you want to be really safe, I can ensure there is always a `/` added to `GetBasePath` here too. I think what would be good would be some kind of function internal to this library for building up URLs. Similar to how Flurl does it for example.

Reply:
Ok, looks good to me, assuming that GetBasePath() will return either /, null or ""..